### PR TITLE
Fix migration crash and bump reanimated to more stable version

### PR DESCRIPTION
**Pull request by Perronef5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -159,7 +159,7 @@
     "react-native-pager-view": "6.0.1",
     "react-native-qrcode-svg": "6.1.2",
     "react-native-randombytes": "3.6.1",
-    "react-native-reanimated": "2.12.0",
+    "react-native-reanimated": "2.14.4",
```
Comment (tyler-whitman) [inline package.json:162]: Hmm I remember having an issue with 2.14. Did you try v3?

Comment (tyler-whitman) [reply package.json:162]: Maybe they fixed the issue I was having in one of the 2.14 patches

**Review by tyler-whitman — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Perronef5) [reply package.json:162]: Hmmm let me check Android. V3 is giving me a bunch of build issues :/

Comment (tyler-whitman) [reply package.json:162]: No problem, I think the 2.14 issue I had was build issues too so if it’s building it’s probably fine

**Pull request merged**